go.mod

```diff
@@ -77,7 +79,8 @@ require (
 	github.com/ory/jsonschema/v3 v3.0.8
 	github.com/ory/mail/v3 v3.0.0
 	github.com/ory/nosurf v1.2.7
-	github.com/ory/x v0.0.590
+	github.com/ory/x v0.0.591-0.20231009141749-f6c175b9bd1a
```

Comment:
Bump to official version, PR is merged already.

Reply:
done